.github/workflows/release.yaml

```
@@ -306,6 +310,13 @@ jobs:
           name: package-${{ matrix.platform }}
           retention-days: 1
 
+      - name: Upload binary to workflow artifacts for GitHub Pages (web)
```

```suggestion
      - name: Upload web bundle to workflow artifacts for GitHub Pages (web)
```
Okay I see now, `bevy build web --bundle` gives the bundle the same name as the `cargo build` binary name, so here you're uploading the whole web bundle as an artifact 👍

Some thoughts out of scope for this PR: I wonder if we should be renaming the bundle folder to something else (like `app_binary_name` at least), but I suppose unlike the packages for other platforms, players don't download and unzip the package themselves so they never see the names.